dipy/denoise/tests/test_nlmeans.py

```diff
@@ -90,8 +90,15 @@ def test_nlmeans_4d_3dsigma_and_threads():
     sigma = np.ones(data.shape[:3])
     mask = np.zeros(data.shape[:3])
 
-    # mask[25-10:25+10] = 1
-    mask[:] = 1
+    # If you don't provide large enough datasets the OS or OpenMP may not give
+    # all threads to all CPUs
+    # We use False by default to make the test run faster.
+    heavy_duty = False
```

Comment:
No, that would be a bad idea because the point is to make the test faster in all computers both Travis and ours. But if someone wants to see the effect of his cpus being used he can change the variable. To see if you are using all your cpus you need to check manually your system monitor. This is quite complicated to do automatically.  I hope this is clear. :)